### ISSUE TITLE
Add system diagnostics console

### ISSUE THREAD

**Issue by podhrmic** [GaloisInc/BESSPIN-Tool-Suite#1062]
It is a command line interface, that lets you to ask questions about the demonstrator. Runs on the SimPC, start automatically. Can be a part of Ignition (starts from the same terminal window), but doesn't have to be.

It prints errors/debug info on the terminal, and right now has one command - `info`:

```
(ignition ?
Documented commands (type help <topic>):
========================================
help info
```

If you type in `info` it will print the current health of the system/status of individual components (based on #1254).

**Comment by podhrmic** [comment]
So far a proxy for the debug console is the ssh access to the AdminPC (with several tmux windows). Iff this can be properly piped/run from the Sim PC, then a terminal on SimPC might be sufficient.

**Comment by podhrmic** [comment]
Fixed in #1274
